services/feedz/feedz.service.js

```diff
@@ -23,6 +24,18 @@ const schema = Joi.object({
     .default([]),
 }).required()
 
+const singlePageSchema = Joi.object({
+  items: Joi.array()
+    .items(
+      Joi.object({
+        catalogEntry: Joi.object({
+          version: Joi.string().required(),
+        }).required(),
+      })
+    )
+    .default([]),
```

Comment:
What would you think about creating a separate schema for this, e.g. `itemsSchema` and then reusing it in the main two schemas? That should help make things a bit more succinct and encapsulate the definition in one place

Reply:
Yes, you're right. I can also go a little bit further and re-use the `singlePageSchema` as that is the type of object inside the first `items` array. Missed that fact earlier.